proto/spire/api/types/federationrelationship.proto

```diff
@@ -20,17 +20,25 @@ message FederationRelationship {
         // Use SPIFFE Authentication endpoint profile.
         HTTPSSPIFFEProfile https_spiffe = 4;
     }
+
+    // Optional. The bundle for the trust domain. This field can be used to
+    // create/replace the bundle when the relationship is created or updated.
+    // When the relationship is retrieved, it will be set to the latest trust
```

Comment:
nit: which trust domain? ours or theirs? i know it sounds silly in the context, but the `bundle` term can never be harmed by over-clarification

```suggestion
    // When the relationship is retrieved, it will be set to the referenced trust domain's latest bundle
```